### DISCUSSION TITLE
⚠️ POLL ⚠️ What do you use NeuroKit for?

### DISCUSSION THREAD

**Discussion by DominiqueMakowski** [neuropsychology/NeuroKit#884] [Polls]
This poll will help us understand the needs and priorities of the community. Help sharing so that we get an accurate view!

**Comment by Mario-0** [comment]
many people use neurokit for multiple signals, so this question should be a checkbox, not radio button

**Comment by helderneves91** [comment]
Actually I'm starting to test ECG.
I think the package is laking a feature (I don't find it). ECG need to have a grid to analyze it, a line with a white background doesn't mean anything. Am I missing something? Attaching examples.

![exam_id_2_layout_3by4plus2](https://github.com/user-attachments/assets/8f5bae1a-daed-441f-a734-13a11327a893)
![exam_id_2_layout_12by1](https://github.com/user-attachments/assets/1bdc2bbe-47f9-42f6-af6b-f23b2941a8c1)

**Comment by Nishargo** [comment]
The top results say ECG, PPG, EDA, and other. I wonder what 'other' refers to. Also wondering if the results would have changed during the time span of NeuroKit. Maybe another poll might produce new results.
